java/configuration/src/main/java/sleeper/configuration/properties/table/S3TableProperties.java

```diff
@@ -51,8 +53,12 @@ public TableProperties loadProperties(TableIdentity tableId) {
         String bucket = instanceProperties.get(CONFIG_BUCKET);
         String key = getS3Key(tableId);
         LOGGER.info("Loading table properties from bucket {}, key {}", bucket, key);
-        String content = s3Client.getObjectAsString(bucket, key);
-        return new TableProperties(instanceProperties, PropertiesUtils.loadProperties(content));
+        try {
+            String content = s3Client.getObjectAsString(bucket, key);
+            return new TableProperties(instanceProperties, PropertiesUtils.loadProperties(content));
+        } catch (AmazonS3Exception e) {
+            throw TableNotFoundException.withTableIdentity(tableId);
```

Comment:
This can retain the AmazonS3Exception as the cause.

Reply:
Thanks, this exception is now being stored as the cause of the new exception